modalapi/wifi.py

```diff
@@ -117,12 +117,46 @@ def enable_hotspot(self):
         try:
             subprocess.check_output(['sudo', 'systemctl', 'enable', 'wifi-hotspot']).strip().decode('utf-8')
             subprocess.check_output(['sudo', 'systemctl', 'start', 'wifi-hotspot']).strip().decode('utf-8')
+            self.wifi_status = True
         except:
             logging.debug('Wifi hotspot enabling failed')
+            self.wifi_status = False
 
     def disable_hotspot(self):
         try:
             subprocess.check_output(['sudo', 'systemctl', 'stop', 'wifi-hotspot']).strip().decode('utf-8')
             subprocess.check_output(['sudo', 'systemctl', 'disable', 'wifi-hotspot']).strip().decode('utf-8')
+            self.wifi_status = False
         except:
             logging.debug('Wifi hotspot disabling failed')
+            self.wifi_status = True
+
+    def configure_wifi(self, ssid, password):
+        config_lines = [
+            'ctrl_interface=DIR=/var/run/wpa_suplicant GROUP=netdev',
+            'update_config=1',
+            'country=US',
+            '\n'
+            'network={',
+            '\tssid="{}"'.format(ssid),
+            '\tpsk="{}"'.format(password),
+            '}'
+        ]
+        config = '\n'.join(config_lines)
+
+        os.popen("sudo chmod a+w /etc/wpa_supplicant/wpa_supplicant.conf")
```

Comment:
This file path is referenced in 3 places, too easy to miss one or typo, if it ever changes.  Please declare it in one place and then reference the variable each place.